kotest-framework/kotest-framework-plugin-gradle/src/main/kotlin/io/kotest/framework/gradle/KotestPlugin.kt

```diff
@@ -155,6 +167,33 @@ abstract class KotestPlugin : Plugin<Project> {
       }
    }
 
+   //TODO wire this to JVM test Sources automatically
+   private fun KotlinDependencyHandler.addKotestJvmRunner() {
+      project.logger.info("  Adding Kotest JUnit runner")
+      implementation("io.kotest:kotest-runner-junit5-jvm")
```

Comment:
I don't think we should do this as in Kotest 6, using junit5 runner is optional and will be discouraged in favour of our more powerful gradle plugin.

Reply:
so the custom report generator will be used by default?